### ISSUE TITLE
I cannot get good loss with yolov4-mish compared with yolov4.

### ISSUE THREAD

**Issue by shigeomi-t** [AlexeyAB/darknet#7131]
I tried yolov4x-mish, but could not get good results.  
I use my own data to learn and my losses do not go down from around 20. With YOLOv4, the loss is below 1 at about 5000 iterations, and good detection accuracy can be obtained. I use the same training data and the same PC environment when training with yolov4 and yolov4x-mish.

Why is it that yolov4x-mish is less accurate than yolov4? Am I doing something wrong?
Can anyone give me some ideas?

This is a chart of training my data with yolov4-mish.

![localhost](https://user-images.githubusercontent.com/46471121/102225261-5a3c2700-3f2a-11eb-8249-5a4223f7f818.jpg)

- I pulled latest code at 2020-12-15 23:00 JST.
- I changed `classes=1`, `filters=18`
- my conf `yolov4x-mish.cfg`
  ```
  [net]
  batch=64
  subdivisions=8
  width=416
  height=416
  channels=3
  momentum=0.949
  decay=0.0005
  angle=0
  saturation = 1.5
  exposure = 1.5
  hue=.1

  learning_rate=0.001
  burn_in=1000
  max_batches = 300000
  policy=steps
  steps=240000,270000
  scales=.1,.1

  mosaic=0
  ...
  ```
- I have no `bad.list` and `bad_label.list` .
- my cuda info
  ```
   CUDA-version: 10020 (10020), cuDNN: 7.6.5, GPU count: 1   
   OpenCV version: 4.3.0
   0 : compute_capability = 610, cudnn_half = 0, GPU: GeForce GTX 1080 Ti  
   layer   filters  size/strd(dil)      input                output
  ```

**Comment by AlexeyAB** [comment]
Training goes well.
Look at the accuracy (mAP) instead of Loss.

**Comment by shigeomi-t** [comment]
@AlexeyAB 
Thanks for your reply. I'll look at the mAP.
However, using the same data set, YOLOv4x-mish yielded mAPs of 81% at best, while YOLOv4 yielded mAPs of 86% at best. I wonder why YOLOv4x-mish has a lower mAP than YOLOv4.
Doesn't YOLOv4x-mish work well for simple object detection tasks? For example, the task of object detection for human hands only. Is the model too large for the object detection task?
I want to benefit from the improved accuracy of yolov4x-mish.

**Comment by AlexeyAB** [comment]
* YOLOv4x-mish - should be trained longer. Also if you use 416x416 or 512x512, then try to use anchors from yolov4.cfg.

* YOLOv4x-mish improves only `~+1% AP50` and `~+5% AP50...95` if you use the same 512x512 resolution as yolov4.cfg. So it will detect approximately the same number of objects, but with more accurate coordinates.

* YOLOv4x-mish  is more suitable for high resolutions 640x640 - 832x832 than yolov4.cfg, so you will get more than +1% AP50 improvement.

**Comment by shigeomi-t** [comment]
I understood why I can't get better mAP. I will use YOLOv4x-mish when I want to train 640x640 - 832x832 images, and YOLOv4 when the training data is low enough resolution like 416px.

I've received enough answers so I close this issue.
Thank you very much.

**Comment by akashAD98** [comment]
> * YOLOv4x-mish - should be trained longer. Also if you use 416x416 or 512x512, then try to use anchors from yolov4.cfg.
> * YOLOv4x-mish improves only `~+1% AP50` and `~+5% AP50...95` if you use the same 512x512 resolution as yolov4.cfg. So it will detect approximately the same number of objects, but with more accurate coordinates.
> * YOLOv4x-mish  is more suitable for high resolutions 640x640 - 832x832 than yolov4.cfg, so you will get more than +1% AP50 improvement.

1. youv4.cfg & yolov4x-mish both having the same anchors ?? why you said use yolov4 anchors ? can you please elaborate on this ?thanks
